core/src/main/java/org/elasticsearch/rest/action/cat/RestAliasAction.java

```diff
@@ -46,7 +46,7 @@ public RestAliasAction(Settings settings, RestController controller) {
     @Override
     protected RestChannelConsumer doCatRequest(final RestRequest request, final NodeClient client) {
         final GetAliasesRequest getAliasesRequest = request.hasParam("alias") ?
-                new GetAliasesRequest(request.param("alias")) :
+                new GetAliasesRequest(request.param("alias").split(",")) :
```

Comment:
you could use `Strings.commaDelimitedListToStringArray` here as we do in other places.